### ISSUE TITLE
Please read

### ISSUE THREAD

**Issue by samyy5** [gkbrk/slowloris#54]
This is what he writes when I confirm that he will start the attack
 Sending keep-alive headers
Socket count: 0
Well it must have sent more than zero.
Did I do something wrong?

**Comment by DioBuon** [comment]
Yes, you can do the attack only if the site is like that: example.com or xxxx.it, or xxxx.ca.
